config/openstack.toml

```diff
@@ -0,0 +1,31 @@
+[default]
+
+# The node image by either ID or NAME
+os_node_image = "openSUSE-Leap-15.1-OpenStack-201905"
+
+# The node size or flavour name as known by the provider
+os_node_size = "m1.medium"
+
+# libcloud settings:
+os_auth_version = "@jinja {{env['OS_AUTH_VERSION'] or '3.x_password'}}"
+os_verify_ssl_cert = true
+
+# The following settings can be sourced from your openrc v3
+# NOTE(jhesketh): libcloud doesn't load clouds.yaml, so env vars must be set.
+
+os_auth_url = "@jinja {{env['OS_AUTH_URL']}}"
+
+# NOTE(jhesketh): OS_PROJECT_ID is not supported
+
+os_project_name = "@jinja {{env['OS_PROJECT_NAME'] or 'default'}}"
```

Comment:
Doesn't `dynaconf` automatically provide config via env vars? Like https://dynaconf.readthedocs.io/en/latest/guides/environment_variables.html . I guess that would be incompatible naming so we can not use a downloaded openrc file?